desktop/apps/categories2/components/Sidebar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+class Sidebar extends React.Component {
+  constructor(props){
+    super(props)
+  }
+
+  items(){
+    const items = this.props.data
+    return items.map((value, index) => {
+      return <li key={`gene-${index}`}>{value.name}</li>
```

Comment:
The fixture data includes the BSON IDs, so maybe we could just start using `value.id` already as the `key`?